lib/resolvers/plugin_query/admin-plugin-query.js

```diff
@@ -1,29 +1,29 @@
-const Plugin = require('../../models/Plugins');
-const adminCheck = require('../functions/adminCheck');
-const Organization = require('../../models/Organization');
+// const Plugin = require('../../models/Plugins');
+// const adminCheck = require('../functions/adminCheck');
+// const Organization = require('../../models/Organization');
 
-const { NotFoundError } = require('errors');
-const requestContext = require('talawa-request-context');
+// const { NotFoundError } = require('errors');
+// const requestContext = require('talawa-request-context');
 
-module.exports = async (parent, args, context) => {
-  const organizationFound = await Organization.findOne({
-    _id: args.orgId,
-  });
+// module.exports = async (parent, args, context) => {
+//   const organizationFound = await Organization.findOne({
+//     _id: args.orgId,
+//   });
 
-  if (!organizationFound) {
-    throw new NotFoundError(
-      requestContext.translate('organization.notFound'),
-      'organization.notFound',
-      'organization'
-    );
-  }
+//   if (!organizationFound) {
+//     throw new NotFoundError(
+//       requestContext.translate('organization.notFound'),
+//       'organization.notFound',
+//       'organization'
+//     );
+//   }
 
-  adminCheck(context, organizationFound);
-  const pluginFound = await Plugin.find({
-    orgId: args.orgId,
-    pluginStatus: 'ACTIVE',
-    adminAccessAllowed: true,
-  });
+//   adminCheck(context, organizationFound);
+//   const pluginFound = await Plugin.find({
+//     orgId: args.orgId,
+//     pluginStatus: 'ACTIVE',
+//     adminAccessAllowed: true,
+//   });
 
-  return pluginFound;
-};
+//   return pluginFound;
+// };
```

Comment:
Why comment out the entire file and not remove it?

Reply:
I'll change these right away